tutorial/SparkRedshiftTutorial.scala

```diff
@@ -0,0 +1,172 @@
+			.load()
+		eventsDF.show()
+		eventsDF.printSchema()
+
+    /** Load from a query */
+		val salesQuery = """select salesid,listid,sellerid,buyerid,
+                        eventid,dateid,qtysold,pricepaid,commission 
+                        from sales 
+                        order by saletime desc LIMIT 10000"""
+		val salesDF = sqlContext.read
+			.format("com.databricks.spark.redshift")
+			.option("url", jdbcURL)
+			.option("tempdir", tempS3Dir)
+			.option("query", salesQuery)
+			.load()
+		salesDF.show()
+
+		val eventQuery = "select * from event"
+		val eventDF = sqlContext.read
+			.format("com.databricks.spark.redshift")
+			.option("url", jdbcURL) 
+			.option("tempdir", tempS3Dir)
+			.option("query", eventQuery)
+			.load()
+
+    /** Register 'event' table as 'myevent' in the Spark Environment */
+		eventsDF.registerTempTable("myevent") 
+
+    /** Save to a redshift table from a table registered in spark */
+    
+    /*
+     * Create a new table redshiftevent after dropping any existing redshiftevent table
+     * and write event records with event id less than 1000
+     */
+		sqlContext.sql("select * from myevent where eventid<=1000").withColumnRenamed("eventid", "id")
+			.write.format("com.databricks.spark.redshift")
+			.option("url", jdbcURL)
+			.option("tempdir", tempS3Dir)
+			.option("dbtable", "redshiftevent")
+			.mode(SaveMode.Overwrite)
+			.save()
+    
+    /*
+     * Append to an existing table redshiftevent if it exists or create a new one if it does not
+     * exist and write event records with event id greater than 1000
+     */
+		sqlContext.sql("select * from myevent where eventid>1000").withColumnRenamed("eventid", "id")
+			.write.format("com.databricks.spark.redshift")
+			.option("url", jdbcURL) 
+			.option("tempdir", tempS3Dir)
+			.option("dbtable", "redshiftevent")
+			.mode(SaveMode.Append) 
+			.save()
+
+		/** Demonstration of Inter-operability */
+		val salesAGGQuery = """select sales.eventid as id,sum(qtysold) as totalqty,sum(pricepaid)  as salesamt
+                           from sales
+                           group by (sales.eventid)
+                           """
+		val salesAGGDF = sqlContext.read
+			.format("com.databricks.spark.redshift")
+			.option("url", jdbcURL) 
+			.option("tempdir", tempS3Dir) 
+			.option("query", salesAGGQuery) 
+			.load()
+		salesAGGDF.registerTempTable("salesagg")
+
+    /*
+     * Join two DataFrame instances. Each could be sourced from any 
+     * compatible Data Source
+     */
+		val salesAGGDF2 = salesAGGDF.join(eventsDF, salesAGGDF("id") === eventsDF("eventid"))
+			.select("id", "eventname", "totalqty", "salesamt")
+
+    /** Create a custom schema */
+		val schema = StructType(List(StructField("eventid", IntegerType, false),
+			StructField("eventname", StringType, false),
+			StructField("qtysold", LongType, false),
+			StructField("revenue", DecimalType(38, 2), false)))
+      
+    /** Apply custom schema on existing DataFrame */
```

Comment:
This custom schema is essentially just changing the nullability and names of fields, but not the types, right? I wonder if this is going to be confusing to readers because it's not demonstrating a `spark-redshift` feature itself, per-se.